docs/docker-stack/docker-images-recipes/go-beam.Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+ARG BASE_AIRFLOW_IMAGE
+FROM ${BASE_AIRFLOW_IMAGE}
+
+SHELL ["/bin/bash", "-o", "pipefail", "-e", "-u", "-x", "-c"]
+
+USER 0
+
+ENV GO_INSTALL_DIR=/usr/local/go
+
+# Install Go
+RUN DOWNLOAD_URL="https://dl.google.com/go/go1.16.4.linux-amd64.tar.gz" \
```

Comment:
Can you extract the version to the argument to keep consistency with other recipes?
https://github.com/apache/airflow/blob/65871e36f326414082b5b1969cfea2ca082254cc/docs/docker-stack/docker-images-recipes/gcloud.Dockerfile#L22
https://github.com/apache/airflow/blob/65871e36f326414082b5b1969cfea2ca082254cc/docs/docker-stack/docker-images-recipes/hadoop.Dockerfile#L39

Reply:
You are right, good catch. Done :)